specification/edgegateway/resource-manager/readme.ruby.md

```diff
@@ -12,15 +12,15 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
-  - tag: package-2018-07
+  - tag: package-2019-03
 ```
 
-### Tag: package-2018-07 and ruby
+### Tag: package-2019-03 and ruby
 
-These settings apply only when `--tag=package-2018-07 --ruby` is specified on the command line.
+These settings apply only when `--tag=package-2019-03 --ruby` is specified on the command line.
 Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
 
-``` yaml $(tag) == 'package-2018-07' && $(ruby)
-namespace: "Azure::Compute::Mgmt::V2018_07_01"
+``` yaml $(tag) == 'package-2019-03' && $(ruby)
+namespace: "Azure::Compute::Mgmt::V2019_03_01"
```

Comment:
Your namespace cannot be Azure::Compute.  (You are not Compute Service r8???) It must be Azure::EdgeGateway.

Reply:
Fixed